Makefile

```diff
@@ -1,39 +1,39 @@
-GIT_CLONE = git clone --depth 1
+# '$(PIP_INSTALL)'
+PIP_INSTALL = pip install
 
-# Docker tag
-DOCKER_TAG = ocrd/calamari
+# '$(GIT_CLONE)'
+GIT_CLONE = git clone --depth 1
 
 # BEGIN-EVAL makefile-parser --make-help Makefile
 
 help:
 	@echo ""
 	@echo "  Targets"
 	@echo ""
-	@echo "    calamari         git clone calamari"
-	@echo "    calamari_models  git clone calamari_models"
-	@echo "    calamari/build   Install calamari"
-	@echo "    docker           Build docker image"
+	@echo "    install          Install ocrd_calamari"
+	@echo "    calamari         Clone calamari repo"
+	@echo "    calamari_models  Clone calamari_models repo"
+	@echo "    calamari/build   pip install calamari"
 	@echo ""
 	@echo "  Variables"
 	@echo ""
-	@echo "    DOCKER_TAG  Docker tag"
+	@echo "    PIP_INSTALL  '$(PIP_INSTALL)'"
+	@echo "    GIT_CLONE    '$(GIT_CLONE)'"
 
 # END-EVAL
 
-# git clone calamari
+# Install ocrd_calamari
+install:
+	$(PIP_INSTALL) .
```

Comment:
Is this the convention in OCR-D? this does `pip install` which invokes a Python 2 pip on my system, and will break because we need Python 3

Reply:
The convention is to install into a venv which should be python3 based and alias `pip` to `pip3`. But it does not hurt to be explicit. See also https://github.com/stweil/ocrd_all/pull/6#discussion_r352092771